pages/dnd/styles.scss

```diff
@@ -35,49 +40,9 @@
   margin-block-end: 0;
 }
 
-.sorting {
-  & > .option-body {
-    border-block-end: 1px solid awsui.$color-border-divider-default;
-  }
-}
-.sorting:last-child {
-  border-block-end: none;
-}
-
-.placeholder {
-  > .container-option,
-  > .option-body,
-  > td {
-    position: relative;
-    &:after {
-      content: ' ';
-      position: absolute;
-      inset: 0;
-      background: awsui.$color-drag-placeholder-hover;
-    }
-  }
-  > .container-option:after {
-    border-start-start-radius: awsui.$border-radius-container;
-    border-start-end-radius: awsui.$border-radius-container;
-    border-end-start-radius: awsui.$border-radius-container;
-    border-end-end-radius: awsui.$border-radius-container;
-  }
-  > .option-body:after {
-    border-start-start-radius: awsui.$border-radius-item;
-    border-start-end-radius: awsui.$border-radius-item;
-    border-end-start-radius: awsui.$border-radius-item;
-    border-end-end-radius: awsui.$border-radius-item;
-  }
-  > td:first-child {
-    border-start-start-radius: awsui.$border-radius-item;
-    border-end-start-radius: awsui.$border-radius-item;
-  }
-  > td:last-child {
-    border-start-end-radius: awsui.$border-radius-item;
-    border-end-end-radius: awsui.$border-radius-item;
-  }
+.row {
+  border-block-end: 1px solid transparent;
```

Comment:
Why is this rule needed? Removing it in the browser dev tools doesn't seem to make any difference.

Reply:
Without this style the placeholder would overlap the bottom border

<img width="1696" alt="Screenshot 2024-12-18 at 15 44 03" src="https://github.com/user-attachments/assets/d9c069de-26a3-4fa9-85e2-0e8c9715bf87" />